README.md

```diff
@@ -35,14 +35,76 @@ uv run evaluate_encoder_processor_decoder \
 	--config-path=configs/ \
 	--work-dir=outputs/processor_eval \
 	--checkpoint=outputs/encoder_processor_decoder_run/encoder_processor_decoder.ckpt \
-	--batch-index=0 --batch-index=3 \
+	--batch-index=0 \
+	--batch-index=1 \
 	--video-dir=outputs/encoder_processor_decoder_run/videos
 ```
 
 Evaluation writes a CSV of aggregate metrics to `--csv-path` (defaults to
 `<work-dir>/evaluation_metrics.csv`) and, when `--batch-index` is provided,
 stores rollout animations for the specified test batches.
 
+## Example pipeline
+
+This assumes you have the `reaction_diffusion` dataset stored at the path specified by
+the `AUTOCAST_DATASETS` environment variable.
+
+### Train autoencoder
+```bash
+uv run python -m autocast.train.autoencoder \
```

Comment:
This requires having dev dependencies installed (it failed for me because I didn't have `beartype`) so this either shouldn't be a dev dependancy or we need to note this somewhere